src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -85,47 +88,47 @@ public void LogFunctionExecutionEvent(string executionId, string siteName, int c
             }
 
             [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational)]
-            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string eventCreatedAt)
+            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string timeCreated)
```

Comment:
Also, question on column names - for these new "dynamic" columns we're adding, how does casing work? I want them to be pascal cased in the actual tables, but these params are camel cased.